Build fails -

After a rebase from master, the build fails with:

````
Requirement already satisfied (use --upgrade to upgrade): WebOb==1.4 in ./.build/venv/lib/python2.7/site-packages (from -r CONST_versions.txt (line 74))
Requirement already satisfied (use --upgrade to upgrade): zope.component==4.2.1 in ./.build/venv/lib/python2.7/site-packages (from -r CONST_versions.txt (line 75))
Requirement already satisfied (use --upgrade to upgrade): zope.deprecation==4.1.2 in ./.build/venv/lib/python2.7/site-packages (from -r CONST_versions.txt (line 76))
Requirement already satisfied (use --upgrade to upgrade): zope.event==4.0.3 in ./.build/venv/lib/python2.7/site-packages (from -r CONST_versions.txt (line 77))
Requirement already satisfied (use --upgrade to upgrade): zope.interface==4.1.2 in ./.build/venv/lib/python2.7/site-packages (from -r CONST_versions.txt (line 78))
Requirement already satisfied (use --upgrade to upgrade): zope.sqlalchemy==0.7.5 in ./.build/venv/lib/python2.7/site-packages (from -r CONST_versions.txt (line 79))
Obtaining pyramid-closure from git+https://github.com/camptocamp/pyramid_closure#egg=pyramid_closure (from -r CONST_requirements.txt (line 5))
  Updating ./.build/venv/src/pyramid-closure clone
Obtaining pyramid from git+https://github.com/Pylons/pyramid@620bdefbff81792fd047f273a234264ce3d0a764#egg=pyramid (from -r CONST_requirements.txt (line 6))
  Updating ./.build/venv/src/pyramid clone (to 620bdefbff81792fd047f273a234264ce3d0a764)
  Could not find a tag or branch '620bdefbff81792fd047f273a234264ce3d0a764', assuming commit.
^@^@fatal: unable to connect to github.com:
github.com[0: 192.30.252.129]: errno=Connection timed out

Clone of 'git://github.com/Pylons/pylons_sphinx_theme.git' into submodule path 'docs/_themes' failed
  Complete output from command /bin/git submodule update --init --recursive -q:

  ----------------------------------------
  Command "/bin/git submodule update --init --recursive -q" failed with error code 1 in /home/wepa/geoportailv3/.build/venv/src/pyramid
make: *** [.build/requirements.timestamp] Error 1
````

The submodules fails to update, because it is specified by git protocol, not https protocol . Needs to be fixed 

Restant c2c 01/03/2015 : 1j

--- a/CONST_requirements.txt
+++ b/CONST_requirements.txt
@@ -3,5 +3,5 @@
 --find-links http://pypi.camptocamp.net/internal-pypi/index/c2cgeoportal-win
 -r CONST_versions.txt
 -e git+https://github.com/camptocamp/pyramid_closure#egg=pyramid_closure
--e git+https://github.com/Pylons/pyramid@620bdefbff81792fd047f273a234264ce3d0a764#egg=pyramid
+-e git+https://github.com/Pylons/pyramid@1e02bbfc0df09259bf207112acf019c8dba44a90#egg=pyramid
 -e .

